chainer/links/model/vision/resnet.py

```diff
@@ -221,6 +244,150 @@ def predict(self, images, oversample=True):
         return y
 
 
+class ResNet50Layers(ResNetLayers):
+
+    """A pre-trained CNN model with 50 layers provided by MSRA [1].
```

Comment:
Please describe the details of the ResNet50Layers. What is the advantage of ResNet50Layers compared with the other ResNets? When should we use this model? Which is the best to use in the case where the user is unfamiliar with this field?